src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -111,8 +111,8 @@ public function update(): array
      */
     public static function parsePackageName(string $packageName): ?array
     {
-        // https://regex101.com/r/58bl9L/1
-        $regex = '/(?:(?P<registry>[^:\n]+):)?(?P<package>[^@\n]+)(?:@(?P<version>[^\s\n]+))?/';
+        // https://regex101.com/r/d99BEc/1
```

Comment:
I would remove this comment entirely. Someone wanting to get the regex explanation on regex101.com can copy-paste the regex instead.

Reply:
I got this idea from @dunglas - the nice thing about it is that the regex includes a bunch of examples. So if there is a problem in the future, you can quickly open this to see what the passing test cases are and then either validate that (A) the situation you are having trouble with SHOULD be working or (B) that your test case was never included.

Reply:
Exactly, the main benefit it that we can add tests for the regex.